qa/tasks/ceph_fuse.py

```diff
@@ -158,7 +158,7 @@ def task(ctx, config):
             mount_x.cephfs_mntpt = config.get("mount_path")
         if config.get("mountpoint"):
             mount_x.hostfs_mntpt = config.get("mountpoint")
-        mount_x.mount()
+        mount_x.mount(createfs=False)
```

Comment:
@rishabh-d-dave Can you look at this?  I saw the comment at https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/fuse_mount.py#L41 and am wondering if there is a case that relies on the createfs behavior here?

It looks like the fs suite is happy with this change, fwiw (although this is with ``--filter fuse`` so maybe it missed something): https://pulpito.ceph.com/sage-2021-02-01_16:51:53-fs-wip-sage2-testing-2021-01-29-1249-distro-basic-smithi/

Reply:
On very first look I expected that test failures since `createfs` is `True` by default in `FuseMount.mount()` and `LocalFuseMount.mount()`.  So, I am unsure that lack of failures is due to some YAML changes or my lack of full understanding of QA code for CephFS.

I did a little investigation. It turns out that `createfs` never affects `create` attribute of `Filesystem` or `LocalFilesystem` which explains why this change did not lead to any test failures. The value of `createfs` decides whether `setupfs()` is called. And `setupfs()` always instantiates `Filesystem/LocalFilesystem` without changing the default value of `create` (which is `False`). I confirmed this by running a test in test_cephfs_shell.py using vstart_runner.py after hardcoding `createfs` to `False` in `LocalFuseMount.mount()`. The test passed which confirmed that value of createfs really doesn't matter.

I don't deal with YAML modification mostly while working with QA, so I want a little confirmation: the YAML changes here only affect the test environment for cephadm tests (and not the test environment for the rest of CephFS tests), right?

Apologies for late reply, I don't know how did I miss the github notification for this PR.

Reply:
Correct--the yaml changes are adding a new cephadm+fs test; I didn't touch the other fs tests.  Just the ceph_fuse.py change.

Reply:
Okay. Thanks!

Reply:
`createfs` can probably go and/or needs to be aligned with Filesystem/LocalFilesystem `create`. I'll look into that tomorrow.